graphene/src/layers/style/mod.rs

```diff
@@ -86,23 +101,35 @@ impl Gradient {
 			.map(|(i, entry)| entry.to_string() + if i == 5 { "" } else { "," })
 			.collect::<String>();
 
-		let _ = write!(
-			svg_defs,
-			r#"<linearGradient id="{}" x1="{}" x2="{}" y1="{}" y2="{}" gradientTransform="matrix({})">{}</linearGradient>"#,
-			self.uuid, start.x, end.x, start.y, end.y, transform, positions
-		);
+		match self.gradient_type {
+			GradientType::Linear => {
+				let _ = write!(
+					svg_defs,
+					r#"<linearGradient id="{}" x1="{}" x2="{}" y1="{}" y2="{}" gradientTransform="matrix({})">{}</linearGradient>"#,
+					self.uuid, start.x, end.x, start.y, end.y, transform, positions
+				);
+			}
+			GradientType::Radial => {
+				let radius = (f64::powi(start.x - end.x, 2) + f64::powi(start.y - end.y, 2)).sqrt();
+				let _ = write!(
+					svg_defs,
+					r#"<radialGradient id="{}" cx="{}" cy="{}" r="{}" gradientTransform="matrix({})">{}</radialGradient>"#,
+					self.uuid, start.x, start.y, radius, transform, positions
+				);
+			}
+		}
 	}
 }
 
 /// Describes the fill of a layer.
 ///
-/// Can be None, a solid [Color], a linear [Gradient], or potentially some sort of image or pattern in the future
+/// Can be None, a solid [Color], a linear [Gradient], a radial [Gradient] or potentially some sort of image or pattern in the future
 #[repr(C)]
 #[derive(Debug, Clone, PartialEq, Serialize, Deserialize)]
 pub enum Fill {
 	None,
 	Solid(Color),
-	LinearGradient(Gradient),
+	Gradient(Gradient),
```

Comment:
Since we are storing the linear vs radial type in the `Gradient`, it was unnecessary to differentiate between the two here. We're open to better naming conventions!

Reply:
Resolving for now and rewriting this comment after the git rebase